src/azure-cli/azure/cli/command_modules/privatedns/commands.py

```diff
@@ -12,17 +12,17 @@
 def load_command_table(self, _):
 
     network_privatedns_zone_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.privatedns.operations.private_zones_operations#PrivateZonesOperations.{}',
+        operations_tmpl='azure.mgmt.privatedns.operations#PrivateZonesOperations.{}',
```

Comment:
Now the file name is changed from `private_zones_operations` to `_private_zones_operations`, so remove this unnecessary subitem in module path. You can find the name in SDK source code.